src/components/PersistentBar/PersistentBar.tsx

```diff
@@ -0,0 +1,192 @@
+                            {actionButtonOneProps && (
+                                <SecondaryButton {...actionButtonOneProps} />
+                            )}
+                            {actionButtonTwoProps && (
+                                <SecondaryButton {...actionButtonTwoProps} />
+                            )}
+                            {actionButtonThreeProps && (
+                                <SecondaryButton {...actionButtonThreeProps} />
+                            )}
+                        </>
+                    );
+                case PersistentBarType.bottomBarButtonsOnLeft:
+                    return (
+                        <>
+                            {actionButtonOneProps && (
+                                <DefaultButton {...actionButtonOneProps} />
+                            )}
+                            {actionButtonTwoProps && (
+                                <DefaultButton {...actionButtonTwoProps} />
+                            )}
+                            {actionButtonThreeProps && (
+                                <PrimaryButton {...actionButtonThreeProps} />
+                            )}
+                        </>
+                    );
+                case PersistentBarType.topBarWithText:
+                    return (
+                        <>
+                            <Icon
+                                path={getIconName()}
+                                classNames={styles.icon}
+                            />
+                            <div>{getTexts()}</div>
+                            <Pagination
+                                total={paginationTotal}
+                                layout={[
+                                    PaginationLayoutOptions.Previous,
+                                    PaginationLayoutOptions.Pager,
+                                    PaginationLayoutOptions.Next,
+                                    PaginationLayoutOptions.Simplified,
+                                ]}
+                                {...paginationArgs}
+                            />
+                        </>
+                    );
+                case PersistentBarType.topBarPagination:
+                    return (
+                        <>
+                            <Pagination
+                                total={paginationTotal}
+                                {...paginationArgs}
+                            />
+                        </>
+                    );
+                default:
+                    break;
+            }
+        };
+
+        return (
+            <div
+                {...rest}
+                className={persistentBarClasses}
+                ref={ref}
+                style={style}
+                role={role}
+            >
+                {buttonMenuProps &&
+                    buttonMenuProps.map((button: Object) => {
+                        if (button.type === ButtonType.Secondary) {
```

Comment:
Not all codepaths return a value. You may either return null after the switch or add a default of no-op to get around this error.